README.rst

```diff
@@ -23,6 +23,25 @@ basic TFTP RFC1350 compliance mode, disabling all TFTP extensions for
 increased compatibility would you encouter any problem with your target
 system.
 
+Usage as a Library
+------------------
+
+pTFTPd TFTP client can also be imported and used within a Python script.
+
+```from ptftpd import tftpclient
```

Comment:
This is not how reStructuredText work for code blocks. See below in the "Installation" section for an example.